backend/db/seeds.rb

```diff
@@ -5,3 +5,24 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+require "faker"
+
+10.times do |n|
+  User.create!(
+    email: Faker::Internet.unique.email,
+    password: Faker::Internet.password(min_length: 8)
+  )
+
+  memo = Memo.create!(
+    title: Faker::Lorem.sentence(word_count:10),
+    content: Faker::Lorem.paragraphs(number: 5)
+  )
+        
+  10.times do |m|
+    Comment.create!(
+    memo_id: memo.id,
+    content:  Faker::Lorem.sentence(word_count:5)
```

Comment:
Nits

インデントを整えましょう！

```suggestion
      memo_id: memo.id,
      content:  Faker::Lorem.sentence(word_count:5)
```